utils/merkletrie/difftree.go

```diff
@@ -448,3 +454,18 @@ func diffDirs(changes *Changes, ii *doubleIter) error {
 
 	return nil
 }
+
+func isIgnoredNode(tree noder.Noder, path noder.Path) bool {
+	in, ok := tree.(*gitignore.MatchNoder)
+
+	return ok && in.PathIgnored(path)
+}
+
+func ignoredNode(tree noder.Noder, path noder.Path) (noder.Path, bool) {
```

Comment:
```suggestion

// findIgnoredNode finds the node that matches the path in the tree. 
// If the node is found and ignored, it returns the node and true.
func findIgnoredNode(tree noder.Noder, path noder.Path) (noder.Path, bool) {
```
Maybe this is more intuitive?